bin/celery_worker.sh

```diff
@@ -1,16 +1,35 @@
-#!/bin/sh
+#!/bin/bash
 
-set -e
+set -euo pipefail
+
+LOGLEVEL=${CELERY_LOGLEVEL:-INFO}
+
+QUEUE=${1:-${CELERY_WORKER_QUEUE:=celery}}
+WORKER_NAME=${2:-${CELERY_WORKER_NAME:="${QUEUE}"@%n}}
 
 # Figure out abspath of this script
 SCRIPT=$(readlink -f "$0")
 SCRIPTPATH=$(dirname "$SCRIPT")
 
-LOGLEVEL=${CELERY_LOGLEVEL:-INFO}
-
 # wait for required services
 ${SCRIPTPATH}/wait_for_db.sh
 ${SCRIPTPATH}/wait_for_rabbitmq.sh
 
-echo "Starting celery worker"
-exec celery --app nrc --workdir src worker -l $LOGLEVEL
+# build up worker options array
+worker_options=(
+    "-Q$QUEUE"
+    "-n$WORKER_NAME"
+    "-l$LOGLEVEL"
+    "-Ofair"
+)
+
+if [[ -n "${CELERY_WORKER_CONCURRENCY}" ]]; then
```

Comment:
when I run this script locally I have error

```
bin/celery_worker.sh: line 26: CELERY_WORKER_CONCURRENCY: unbound variable
```

Reply:
good that you checked - I checked with docker-compose and that one has it specified. I'll fix it!